src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  user: User | null;
+  userId: number;
```

Comment:
The 'userId' prop is unnecessary. You are already passing the 'user' object which should contain the user's id.